tmt-app/src/App.test.js

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { render, screen } from "@testing-library/react";
+import { render} from "@testing-library/react";
```

Comment:
Is this file still relevant? can we just remove this if we don't plan to write out any tests right now?

Reply:
No, we do not need it right now so it could be deleted